inline_scan.sh

```diff
@@ -531,17 +535,12 @@ save_and_copy_images() {
     local base_image_name=$(echo ${FULLTAG} | rev | cut -d '/' -f 1 | rev )
     echo "Saving ${base_image_name} for local analysis"
     save_file_name="${base_image_name}.tar"
-    local save_file_path="${VOLUME_PATH}/${save_file_name}"
-
-    docker save "${SCAN_IMAGES[0]}" -o "${save_file_path}"
+    local save_file_path="${TMP_PATH}/${save_file_name}"
 
-    echo "docker save is in progress..."
-    while [[ ! -s "${save_file_path}" ]]; do
-        if [[ "${V_flag:-}" ]]; then
-            echo "waiting for docker save to finish"
-        fi
-        sleep 1
-    done
+    # Eventually remove localbuild from FULLTAG is present or docker save crashes
+    local save_img=$(echo ${FULLTAG} | sed "s/localbuild\///g")
+    docker save "${save_img}" -o "${save_file_path}"
```

Comment:
FYI @haresh-suresh 
It happens that `docker save some-image` (note: no tag) takes all possible tags present in the current machine/local registry, and then skopeo fails during anchore evaluation (because it sees two differents entry in the manifest and can't choose).

FULLTAG is already filled with tags/repos etc, we have just to strip (eventually) `localbuild`